README.md

```diff
@@ -69,15 +66,44 @@ qemu-system-aarch64 \
     -snapshot output/qcow2/disk.qcow2
 ```
 
-## Volumes
-- `/output` - used for output files
-- `/store` - used for the osbuild store
-- `/rpmmd` - used for the dnf-json rpm metadata cache
+## 📝 Arguments
```

Comment:
I think this section would be clearer if it started with a standard manpage style usage line. From just reading the README I am not sure what arguments are required or what their positions should be.

Reply:
I put there something similar to what you proposed - basically what cobra generates for `--help`. Is it better better? I had the similar feeling tbh. The positional argument is important, yet it felt quite hidden.